content/reference/cpp/cpp-generated.md

```diff
@@ -1048,6 +1084,10 @@ In addition, it will generate these methods:
 
 ## Enumerations {#enum}
 
+**Note:** As of edition 2024
```

Comment:
This one is intentionally 2024, not 2023 like the others above?

Reply:
@mkruskal-google says, "Nope, that was intentional. That's a different feature.